src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -3927,6 +3927,23 @@ internal uint GetValEscape(BoundExpression expr, uint scopeOfTheContainingExpres
                             isRefEscape: false);
                     }
 
+                    if (conversion.Conversion.IsUserDefined)
+                    {
+                        var operatorMethod = conversion.Conversion.Method;
+                        Debug.Assert(operatorMethod is not null);
+
+                        return GetInvocationEscapeScope(
```

Comment:
Couldn't come up with any interesting scenarios - it seems ref struct needs to be a return type of the operator to be involved in ref safety checks; and that cannot participate in a standard implicit conversion since it cannot be boxed nor converted to a type parameter.

Reply:
> Is this conversion going to be represented as an additional conversion node on top of the one that we are analyzing?

I am still interested in the answer for this question. We are thinking about supporting standard conversions for span types. I want to make sure the code is ready to deal with the situation that is likely to arise in the future.

Reply:
Yes, the standard implicit conversion is represented on top of the user defined conversion in the bound tree.